openfisca_aotearoa/tests/social_security/home_help.yaml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: "Home Help"
+  period: 2018-08
+  absolute_error_margin: 0
+  persons:
+
+    - id: "NZ-Citizen"
+      is_citizen_or_resident: true
```

Comment:
you shouldn't pass this is. you can pass `is_nz_citizen`, or `is_resident` or `is_permanent_resident`